lib/testing/src/lib/core/pages/form/formFields.ts

```diff
@@ -33,6 +33,8 @@ export class FormFields {
     selectFormContent = element(by.css('div[class*="mat-select-panel"]'));
     completeButton = element(by.id('adf-form-complete'));
     errorMessage = by.css('.adf-error-text-container .adf-error-text');
+    formCloudEditor = element.all(by.css('.mat-tab-list .mat-tab-label')).get(1);
```

Comment:
These 2 locators are related only to demo-shell, so a new page should be created in e2e/pages/demo-shell folder for them. Also these 2 new methods should be moved in there.